sagemaker_processing/fairness_and_explainability/text_explainability_sagemaker_algorithm/byo_blazingtext_model_hosting.ipynb

```diff
@@ -0,0 +1,892 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Running the clarify explainability job involves spinning up a processing job and a model endpoint which may take a few minutes.
</pre><p>Branding nit: capitalize <code>clarify</code> -&gt; <code>Clarify</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
Fixed.